ios/Membrane.swift

```diff
@@ -248,6 +248,15 @@ class Membrane: RCTEventEmitter, MembraneRTCDelegate {
   
   @objc(disconnect:withRejecter:)
   func disconnect(resolve:RCTPromiseResolveBlock, reject:RCTPromiseRejectBlock) -> Void {
+    guard isScreensharingEnabled == false else {
+      DispatchQueue.main.async {
+        RPSystemBroadcastPickerView.show(for: screencastExtensionBundleId)
```

Comment:
this only shows the dialog to the user, an evil user still might choose to keep the screencast on and it's a bad ux
I know the screencast is controlled by the broadcast extension and probably it's the only way to stop the broadcast but have you tried maybe calling `stop` on the screencast track?